make-base/docker-compose.mk

```diff
@@ -17,10 +17,10 @@ images: config-ready docker-compose.yml
 config-ready:: docker-compose.yml
 	@ true
 
-docker-ready:: images
+compose-ready:: images
```

Comment:
clarifies that this target is about preparing dependencies that ar enecessary for docker-compose (so running development servers).